apps/chat/src/utils/app/data/application-service.ts

```diff
@@ -0,0 +1,106 @@
+import { Observable } from 'rxjs';
+
+import {
+  ApplicationDetailsResponse,
+  ApplicationListItemModel,
+  ApplicationListResponseModel,
+  ApplicationMoveModel,
+  CreateApplicationModel,
+} from '@/src/types/applications';
+
+import { ApiUtils } from '../../server/api';
+import { constructPath } from '../file';
+import { BucketService } from './bucket-service';
+
+export class ApplicationService {
```

Comment:
Why didn't you create `ApplicationApiStorage` like `PromptApiStorage` but with `ApiKeys.Applications`?

Reply:
Agreed on make tech dept ticket